src/Microsoft.ML.Data/Model/ModelOperationsCatalog.cs

```diff
@@ -281,6 +302,13 @@ public ITransformer LoadWithDataLoader(string filePath, out IDataLoader<IMultiSt
         /// <typeparamref name="TDst"/>.</param>
         /// <param name="inputSchemaDefinition">Additional settings of the input schema.</param>
         /// <param name="outputSchemaDefinition">Additional settings of the output schema.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[Load](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs)]
```

Comment:
>TextClassification [](start = 103, length = 18)

this looks too complicated. let's use a simpler pipeline with bare minimum (no metrics, no testing, no textloader, just Fit and CreatePredictEngine). You can take one of our simpler examples and cut the unnecessary parts.